_delphi_utils_python/delphi_utils/flash_eval/__main__.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+"""Call the function run_module when executed.
+
+This file indicates that calling the module (`python -m MODULE_NAME`) will
+call the function `run_module` found within the run.py file. There should be
+no need to change this template.
+"""
+
+from .run import run_module
+
+run_module()
```

Comment:
In the interest of keeping the FlaSH package structure as similar as possible to our other packages, I'd recommend switching to this for `__main__` (from the python template) and moving the contents of `flash_eval` into `run_module`.

```suggestion
from .. import read_params
from .run import run_module  # pragma: no cover

run_module(read_params())  # pragma: no cover
```

Reply:
[Python template](https://github.com/cmu-delphi/covidcast-indicators/tree/4641604e5440117fdc800c71220da4bf92f551a0/_template_python) for reference. Check out how the `run.py` file is formatted.

Reply:
Bump this suggested change.